src/javascript/ContentEditor/ContentEditor/EditPanel/HeaderActions/utils.js

```diff
@@ -0,0 +1,16 @@
+import {registry} from '@jahia/ui-extender';
+
+export const getButtonLimitValue = () => {
+    const configValue = contextJsParameters.config.jcontent?.['contentEditorQuickActionsButtons.limit'];
+    const parsedValue = parseInt(configValue, 10);
```

Comment:
default limit value from config different than default here? Maybe make it the same for consistency

Reply:
Also if we can rename utils to utils.headerActions.js instead.